impl/src/main/java/org/ehcache/config/builders/ExpiryPolicyBuilder.java

```diff
@@ -193,13 +193,13 @@ public String toString() {
     }
   }
 
-  private static final class TimeToLiveExpiryPolicy extends BaseExpiryPolicy {
+  public static final class TimeToLiveExpiryPolicy extends BaseExpiryPolicy {
```

Comment:
Can we do this without exposing these classes?

Reply:
It can be, as you suggested.